pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/errorprone/ApexCSRFRule.java

```diff
@@ -17,6 +19,7 @@
  */
 public class ApexCSRFRule extends AbstractApexRule {
     public static final String INIT = "init";
+    private static final String STATIC_INITIALIZER = "<clinit>";
```

Comment:
Sorry if I continue to ask this question but we must get rid of this CodeClimate stuff. I, we, the Salesforce world and you the main maintainer dont want provider specific stuff in PMD. When will it finally be removed?

Reply:
It's still the same plan: We've already deprecated the properties in PMD 6 and will remove them (probably have them already in the branch) in PMD 7.